.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I don't tend to like to add things like these, as I prefer to be tool agnostic. Is this necessary?

Reply:
This isn't used to do anything special and just describes the build command to run. No it's not necessary though. I can add the directory to `.gitignore` if you want.